tools/infer_experiment.cwl

```diff
@@ -0,0 +1,49 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: infer_experiment 
```

Comment:
Why this tool and `tools/alignmentfile_pairedness.cwl`?

Reply:
I think I mean to get rid of this. I originally made this but then when with my own tool because this one takes. a needlessly long time.